src/utilities/fetch-package-repos.js

```diff
@@ -46,13 +48,15 @@ async function main() {
         return item;
       }
 
-      const { organization, suffixes } = item;
+      const { organization, suffixes, hides } = item;
 
       const repos = await paginate(organization);
-
       return repos
         .map(repo => repo.full_name)
-        .filter(name => suffixes.some(suffix => name.endsWith(suffix)));
+        .filter(name =>
+          suffixes.some(suffix => name.endsWith(suffix)) &&
+          !hides.some(hide => name.endsWith(hide))
```

Comment:
I would rather use 2 different filter, but this is fine.
It would also be great to have a test to make sure we are displaying the same number of repos

Reply:
This hide filter isn't scary, we already put them into `loaders.hides[]` and `plugins.hides[]`, so should be pretty safe except for cases when a plugin is called strangely: `super-config-loader` 

Let me change filter to full name instead so there is no risk

Reply:
@montogeek now thats against full match